### ISSUE TITLE
problems with Conv-TasNet speech separation

### ISSUE THREAD

**Issue by Morank88** [mpariente/asteroid#311]
Hi,

I am new to the audio world...

I have tried to use the pretrained model just to explore a simple speech separation example but I have encountered a problem. Here is my code:

import torch
import torchaudio
from asteroid.models import ConvTasNet

# model = ConvTasNet.from_pretrained('mpariente/ConvTasNet_WHAM!_sepclean')
model = ConvTasNet.from_pretrained('Cosentino/ConvTasNet_LibriMix_sep_clean')

model.eval()

# from asteroid.data import LibriMix
# train_set, val_set = LibriMix.mini_from_download(task='sep_clean')

filename = r'/home/MiniLibriMix/train/mix_clean/1025-92820-0032_8410-278217-0015.wav'

torchaudio.set_audio_backend("sox_io")

wav_file, fs = torchaudio.load(filename)

sp = model(wav_file)

sp1 = sp[0,0:1,:]
sp2 = sp[0,1:2,:]

torchaudio.save('sp1.wav', sp1, fs)
torchaudio.save('sp2.wav', sp2, fs)

The problem that I am encountering is that the amplitude of the separated signals sp1 and sp2 are much higher then the amplitude of the input mixed wav, resulting in a distorted separated signals. if I normalize to sp1/10000 and sp2/10000 I get very good signals. What am I missing here? Is there some normalization required to the output?

I appreciate your help.

Best regard
Moran

**Comment by groadabike** [comment]
Hi,
If you check the eval.py file from the librimix/ConvTasNet recipe, the estimations are normalised as 
> est_src *= np.max(np.abs(mix_np)) / np.max(np.abs(est_src))

**Comment by mpariente** [comment]
Current speech separation models use SI-SDR loss, which is _scale invariant_, that's the origin of the problem. 
Closing this because it seems solved.

**Comment by mpariente** [comment]
BTW, please check here next time https://mpariente.github.io/asteroid/faq.html

**Comment by Morank88** [comment]
HI,

Thanks for the quick feedback!!! I'll check it.
